Documentation/project-docs/event-logging-guide.md

```diff
@@ -0,0 +1,25 @@
+#Event Logging in CoreCLR
+
+For event logging mechanism in coreclr, we are using the [ETW](https://msdn.microsoft.com/en-us/library/windows/desktop/bb968803%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)- Event Tracing for Windows as a template for all logging mechanism
```

Comment:
I think I get what you're saying here, but it would be good to clarify the text a bit.  Specifically, it should say that we're using the ETW manifest as the basis for the event definitions on all platforms.  Then, we run various scripts to generate the platform specific code that is used for logging.